docs/node_benchmarks.md

```diff
@@ -0,0 +1,351 @@
+# Node Benchmarks for Scylla
+
+Whether you're running performance jobs or investigating an issue, you might
+want to now if the hardware Scylla is running meets you baseline perfromance
+expectations. That's what node benchmarks are for.
+
+### What benchmarks are we running?
+
+We're running:
+* sysbench - for CPU benchmarking. Github: https://github.com/akopytov/sysbench
+* cassandra-fio - for disk IO benchmarking. Github: https://github.com/KnifeyMoloko/cassandra-fio
+
+### How to run the benchmarks?
+
+Simply add the line `run_node_benchmarks: true` in your test case yaml.
+
+### At what point in the test are the benchmarks performed?
```

Comment:
`At what point in the test are the benchmarks performed?` doesn't sound correct to me
What about this one:
`At which point in time of a test run these benchmarks get performed?`
?

Reply:
Fixed with: `At which point in time of a test run are these benchmarks performed?`